src/utils/isStringEqual.ts

```diff
@@ -0,0 +1,5 @@
+export default function isStringEqual (stringA: string, stringB: string) {
+  return (
+    stringA.localeCompare(stringB, undefined, { sensitivity: 'base' }) === 0
```

Comment:
`localeCompare` looks like an overhead as it checks special characters.
can you just lowercase both strings and compare?